main.go

```diff
@@ -155,7 +160,32 @@ func main() {
 			ServerName:   *serverName,
 		})
 	}
-	db, err := sql.Open("mysql", *dsn)
+
+	var finalDsn = *dsn
+	if *prompt {
+		dsnRegex := regexp.MustCompile(`(\w*):?\w*(@.+)`)
+		matches := dsnRegex.FindStringSubmatch(finalDsn)
+		if matches == nil {
+			fmt.Print("Incorrect format for dsn. Usage:\n")
+			flag.PrintDefaults()
+			os.Exit(1)
+		}
+
+		fmt.Print("Enter password: ")
+		// don't echo password to screen during input
```

Comment:
Please use uppercase for the first work and add a '.' at the end. Same below.

Reference: https://golang.org/doc/effective_go.html#commentary